raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -541,9 +539,13 @@ contract TokenNetwork is Utils {
 
         emit ChannelUnlocked(channel_identifier, participant, unlocked_amount, returned_tokens);
 
-        assert(computed_locksroot != 0);
-        assert(unlocked_amount > 0);
-        assert(locked_amount > 0);
+        // Channel must be settled and channel data deleted
+        require(channels[channel_identifier].state == 0);
+
+        require(computed_locksroot != 0);
+        require(locked_amount > 0);
+        assert(locked_amount >= unlocked_amount);
```

Comment:
Just let me see if I understand correctly. This here can be an `assert` since at [this](https://github.com/raiden-network/raiden-contracts/pull/64/files#diff-2f6fb78ac7aaf29aaec1ed8ab59468f8R522) line we already make sure that `unlocked_amount` is the min of `unlocked_amount` and `locked_amount` so this should always hold true.